sgf-viewer/prepare_data/prepare_data.py

```diff
@@ -121,15 +121,45 @@ def run_cmd(cmd, shell=False, dry_run=False):
                     dict_writer.writeheader()
                     dict_writer.writerows(parsed_games)
 
+            # Matches comments containing winrate, lossrate, tie rate, and score
+            b_regex = re.compile(  # For black
```

Comment:
Do these differ just in B/W? Can probably reduce code duplication by having a base string both are constructed from or a helper method to make these patterns?

Reply:
Done